File: Java/tim.java
public class tim{

    private String namatim;
    private String asal;
    private int tahun;
    //private String pemain;
    private String[] pemain;
    int x = 0;

    public tim(){

    }

    public tim(String namatim, String asal, int tahun, String[] pemain, int m){
        this.namatim = namatim;
        this.asal = asal;
        this.tahun = tahun;
        for(x=0; x<m; x++){
            this.pemain[x]=pemain[x];
        }
    }

    public String getnamatim(){
        return namatim;
    }
    public void setnamatim(String namatim){
        this.namatim = namatim;
    }

    public String getasal(){
        return asal;
    }
    public void setasal(String asal){
        this.asal = asal;
    }
    public int gettahun(){
        return tahun;
    }
    public void settahun(int tahun){
        this.tahun = tahun;
    }
    public String getpemain(int x){
        return pemain[x];
    }
    public void setpemain(String[] pemain, int m){
        for(x=0; x<m; x++){
            this.pemain[x]=pemain[x];
        }
    }


}

File: Python1/tim.py
# deklarasi kelas tim
class tim():

    # deklarasi variabel lokal
    __namatim = ""
    __asal = ""
    __tahun = 0
    __pemain = ""

    # konstruktor
    def __init__(self, namatim="", asal="", tahun=0, pemain=""):
        self.__namatim = namatim
        self.__asal = asal
        self.__tahun = tahun
        self.__pemain = pemain

    # get dan set setiap variabel
    def setnamatim(self, namatim):
        self.__namatim = namatim
    def getnamatim(self):
        return self.__namatim
    def setasal(self, asal):
        self.__asal = asal
    def getasal(self):
        return self.__asal
    def settahun(self, tahun):
        self.__tahun = tahun
    def gettahun(self):
        return self.__tahun
    def setpemain(self, pemain):
        self.__pemain = pemain
    def getpemain(self):
        return self.__pemain